azure-pipelines-contrib.yml

```diff
@@ -38,7 +38,7 @@ stages:
             displayName: 'Use Python $(python.version)'
 
           - script: |
-              pip install isort[requirements]==5.10.1 flake8==5.0.4 black==22.3.0 pyupgrade==2.7.2 invoke==1.7.1
+              pip install $(grep -E '^(black|invoke|isort|flake8|pyupgrade)' reqs/requirements-dev-contrib.txt)
               EXIT_STATUS=0
```

Comment:
This will prevent us from having to hardcode the version numbers in multiple places.

Reply:
Super awesome. Thank you for this